client/App.tsx

```diff
@@ -1,45 +1,33 @@
 import { NavigationContainer } from '@react-navigation/native';
 import { createNativeStackNavigator } from '@react-navigation/native-stack';
-import { useState } from 'react';
 import CategoriesScreen from './src/components/CategoriesScreen';
 import HomeScreen from './src/components/HomeScreen';
 import IntroScreen from './src/components/IntroScreen';
-import { emptyUser } from './src/constants/emptyMocks';
-import { User } from './src/constants/interfaces';
 
 export type StackParamList = {
-  Home: {
-    user: User;
-    setUser: React.Dispatch<React.SetStateAction<User>>;
-  };
-  Categories: {
-    user: User;
-    setUser: React.Dispatch<React.SetStateAction<User>>;
-  };
-  Intro: {
-    setUser: React.Dispatch<React.SetStateAction<User>>;
-  };
+  Home: undefined;
+  Categories: undefined;
+  Intro: undefined;
 };
 
 const Stack = createNativeStackNavigator<StackParamList>();
 
 export default function App() {
-  const [user, setUser] = useState(emptyUser);
 
   return (
     <NavigationContainer>
       <Stack.Navigator>
         <Stack.Screen name="Intro" options={{ headerShown: false }}>
```

Comment:
I believe we may just add `component={IntroScreen}` and props will be accessible